requirements/base.in

```diff
@@ -5,15 +5,16 @@ pytz  # handle timezones
 python-dotenv  # environment variables for secrets
 python-decouple  # processing of envvar configs
 celery
-zgw-consumers
```

Comment:
Hrm, I think I don't agree with this.

This pushes all the config into the process, which may not be what we want. Having the config optional in the process, with defaults in BPTL itself seems like a better approach with the idea of "configure once".

It also complicates usage of the NLX outway, pushing that into processes makes every app starting processes required to be aware of the outway location/pass that, which is simply too much management overhead.